RELEASE_NOTES.md

```diff
@@ -1,5 +1,62 @@
 # Release Notes
 
+## Changes in splinter 0.6.1
+
+### Highlights
+
+* Add a `state migrate` command for migrating state between database backends.
+
+* Update stores to support write-exclusivity for database backends. This fixes
+  a database lock contention issue seen with SQLite deployments.
+
+### libsplinter
+
+* Add validation that no SQL backed trees exist if LMDB is enabled.
+
+* Add `OrchestratableService` to `Orchestrator`.
+
+* Remove lock-contention in `Orchestrator` REST API calls.
+
+* REST API features have been reorganized, placing all Actix Web 1 related code
+  under the rest-api-actix-web-1 related feature.
```

Comment:
Features should be double-quoted and in inline-code quotes

Reply:
Fixed